src/write.rs

```diff
@@ -216,9 +216,8 @@ impl<W> WavWriter<W>
             try!(buffer.write_le_u16(spec.channels));
             // The field nSamplesPerSec.
             try!(buffer.write_le_u32(spec.sample_rate));
-            let bytes_per_sec = spec.sample_rate
-                              * self.bytes_per_sample as u32
-                              * spec.channels as u32;
+            let bytes_per_sec = spec.sample_rate * self.bytes_per_sample as u32 *
+                                spec.channels as u32;
```

Comment:
Please avoid formatting-only changes, they make `git blame` less useful. (Also below.)

You can use `git reset --patch master` to interactively undo these formatting-only changes, and then a `git commit --amend` to update the commit such that these changes never happened.